roles/ipv6ra/tasks/main.yml

```diff
@@ -11,5 +11,10 @@
             mode=0644
   notify: restart-dnsmasq
 
+- name: prevent dnsmasq from add itself to /etc/resolv.conf via resolvconf
+  lineinfile: dest=/etc/default/dnsmasq regexp="^DNSMASQ_EXCEPT="
+              line="DNSMASQ_EXCEPT=lo"
+  notify: restart-dnsmasq
```

Comment:
My ideal here would be to figure out a real strategy for using dnsmasq early on in the process and rely on it being available rather than having it possibly installed later, but that is a bigger conversation. 

I'm not sure that the network hosts vs controller hosts matters. ipv6ra is applied very early on in the site.yml [1]. Ubuntu enables dnsmasq as soon as it is installed leaving us with broken DNS for the next roles.  This is done long before the neutron-data-network role is applied.

I'm still not sure why this doesn't affect ursula today because i cannot perform a run without this change.



[1] https://github.com/blueboxgroup/ursula/blob/master/site.yml#L52